hack/static/cmd/versions.go

```diff
@@ -0,0 +1,174 @@
+package cmd
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/mesosphere/kommander-applications/hack/static/pkg/bloodhound"
+	"github.com/mesosphere/kommander-applications/hack/static/pkg/kapps"
+
+	"github.com/mesosphere/dkp-bloodhound/pkg/parse"
+	"github.com/mesosphere/dkp-bloodhound/pkg/parse/k8sresource"
+	"github.com/mesosphere/dkp-bloodhound/pkg/parse/kommanderapplicationversion"
+
+	"github.com/spf13/cobra"
+	kyaml "sigs.k8s.io/kustomize/kyaml/yaml"
+	syaml "sigs.k8s.io/yaml"
+)
+
+var versionsCmd = &cobra.Command{
+	Use:   "versions",
+	Short: "Output all kommander-applications versions",
+
+	RunE: func(c *cobra.Command, _ []string) error {
+		cursor := bloodhound.Run(branch, tag)
+
+		fmt.Println("Getting chart versions for all applications...")
+		kApps, err := kapps.List(cursor)
+		if err != nil {
+			return err
+		}
+
+		sort.Sort(kApps)
+
+		k, err := cursor.GetByID("kommander-application://kommander")
+		if err != nil {
+			return err
+		}
+
+		fmt.Println("Determining which applications are installed by default...")
+		defaultApps, err := getKommanderDefaultApps(k)
```

Comment:
this will get the enterprise apps and default workspace apps, but those aren't necessarily the entire set of apps that are enabled by default when installing Kommander, maybe we can rename the `Enabled` field to be more precise? 

Here are the default apps deployed by the CLI -> https://github.com/mesosphere/kommander-cli/blob/c283173ea1f3cafdcdfef2ed55ba252d92b2b90c/pkg/installer/config/config.go#L175.

Reply:
Ah, thanks for this. Any idea on how I can get this list of apps by analyzing the kApps repo?

Reply:
@cbuto Is this list reliable for current and future versions? https://github.com/mesosphere/kommander-cli/blob/c283173ea1f3cafdcdfef2ed55ba252d92b2b90c/pkg/installer/config/config.go#L212 if so, I'll just hardcode into this tool.

Reply:
that list could definitely change from release to release, you can see above that where we have a [different set of defaults ](https://github.com/mesosphere/kommander-cli/blob/c283173ea1f3cafdcdfef2ed55ba252d92b2b90c/pkg/installer/config/config.go#L175) for 2.1.

Reply:
It’s hacky and non-trivial, but I could parse the AST. But maybe better to export that map, import it here, and then bump the kommander-cli dependency in this program.

Reply:
Note from today's standup: We'll just not include default apps info here in this tool, and instead that information should be updated in the docs page manually.